jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt

```diff
@@ -815,6 +824,46 @@ class Endpoint @JvmOverloads constructor(
     fun setBandwidthAllocationSettings(message: ReceiverVideoConstraintsMessage) =
         bitrateController.setBandwidthAllocationSettings(message)
 
+    private fun logCurrentSsrcs() {
+        logger.debug {
+            var s = String()
```

Comment:
Should use `buildString`

Reply:
i tried improving this with joinToString.